File: Exemplos M10/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_ReactNative-font/src/intentDigitalHubService/BRIDGE/SetTimeout.ts
import {BridgeCommand} from './BridgeCommand';

export class SetTimeout extends BridgeCommand {
  readonly timeout: number;

  constructor(timeout: number) {
    super('SetTimeout');
    this.timeout = timeout;
  }

  functionParameters(): object {
    return {timeout: this.timeout};
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinAndroid/Elgin_IntentDigitalHUB_XamarinAndroid/IntentServices/Bridge/SetTimeout.cs
namespace Xamarin_Android_Intent_Digital_Hub.IntentServices.Bridge
{
    class SetTimeout : BridgeCommand
    {
        private readonly int timeout;

        public SetTimeout(int timeout) : base("SetTimeout")
        {
            this.timeout = timeout;
        }
        protected override string FunctionParameters()
        {
            return "\"timeout\"" + ":" + timeout;
        }
    }
}